src/isar/storage/blob_storage.py

```diff
@@ -40,13 +40,14 @@ def store(self, inspection: Inspection, metadata: MissionMetadata):
             inspection=inspection, metadata=metadata, filename=local_path.name
         )
 
-        self._upload_file(path=local_path, data=inspection.data)
+        inspection_path = self._upload_file(path=local_path, data=inspection.data)
         self._upload_file(path=local_metadata_path, data=metadata_bytes)
+        return inspection_path
```

Comment:
Double `self._upload_file`, only one needed
```suggestion
        return self._upload_file(path=local_path, data=inspection.data)
```

Reply:
done